### PR TITLE
Add activityCost getter JUnit tests

### LINKED ISSUE #188: Add getter tests to ActivityCost
Add JUnit tests

### PR DESCRIPTION
**Pull request by matheril**
Tests activityCost getter methods.
Fixes #188

### CODE REVIEW

**Review by froststein — CHANGES_REQUESTED** [review]
Consider renaming the test functions to fit with naming conventions for junit test

File: docs/UserGuide.md [hunk 1]
```diff
@@ -8,15 +8,21 @@ activities they participated in, for a particular session, via a Command Line In
...
 
-{Give steps to get started quickly}
-
 1. Ensure that you have Java 11 or above installed.
-1. Down the latest version of `Duke` from [here](http://link.to/duke).
+2. Download the latest .jar version of SplitLah from [here](https://github.com/AY2122S2-CS2113T-T10-1/tp/releases)
+3. Copy the file to the folder you wish to use as a home folder for SplitLah.
+4. Via command line, navigate to the home folder and run it with `java -jar SplitLah.jar`.
+5. Type in a command and press Enter to execute it. A few commands you can try are:
+    - `session /create /n Apple /d 11-03-2022 /pl Warren Ivan Roy` : Creates a Session named "Apple" occurring on
```
Comment (froststein) [inline docs/UserGuide.md:16]: Instead of giving example of "apple" maybe "Outing" would be better.

**Review by froststein — CHANGES_REQUESTED** [review]
Change example in UG

File: docs/UserGuide.md (hunk 1 above)
Comment (matheril) [reply docs/UserGuide.md]: Resolved

**Review by froststein — APPROVED** [review]

**Pull request merged**